pkg/v0/cmd.go

```diff
@@ -351,9 +355,19 @@ func cherryPick(ctx context.Context, logger *logrus.Entry, c internal.Commit, co
 		internal.WithDir(internal.WithEnv(exec.CommandContext(ctx,
 			"go", "mod", "verify",
 		), os.Environ()...), filepath.Join("staging", c.Repo)),
+	}
+
+	manifests := []*exec.Cmd{
 		internal.WithEnv(exec.CommandContext(ctx,
 			"make", "generate-manifests",
 		), os.Environ()...),
+	}
+
+	commits := []*exec.Cmd{
+		// Necessary for untracked files created via `go mod vendor`
+		exec.CommandContext(ctx,
+			"git", "add", "vendor",
```

Comment:
The comment re: untracked makes sense, but let's make this list parallel with the list in `git commit` below, no need to ever have these drift